Fixes #5605 - Remove data withdrawn 2023/03 (West Airspace Deployment)

Liaely opened the pull request:
Fixes #5605 
Fixes #5622 

# Summary of changes

### For #5605:
Note: Fix D076D was not found in Airports/EGGW/Fixes.txt, but was found in Airports/EGKK/Fixes.txt, and in use in Airports/EGKK/Sids.txt in the LAM 5P SID.

Removed withdrawn data (AIRAC 2023/03) from the following files:
- Agreements/Airport/5_FF.txt
- Agreements/Airport/6_FF.txt
- Agreements/Airport/FF_5.txt
- Agreements/Airport/FF_6.txt
- Agreements/Airport/FF_8.txt
- Agreements/Internal/23_25.txt
- Agreements/Internal/23_TCCPT.txt
- Agreements/Internal/23_TCSW.txt
- Agreements/Internal/23_TCVAT.txt
- Agreements/Internal/25_23.txt
- Agreements/Internal/5_S29.txt
- Agreements/Internal/5_Wallasey.txt
- Agreements/Internal/TCCPT_23.txt
- Agreements/Internal/TCSW_23.txt
- Airports/EGCC/Stars.txt
- Airports/EGFF/Fixes.txt (D118E)
- Aiports/EGFF/Sids.txt
- Airports/EGFF/Stars.txt
- Airports/EGGD/Fixes.txt (D087E)
- Airports/EGGD/Sids.txt
- Airports/EGDD/Stars.txt
- Airports/EGGP/Stars.txt
- Airports/EGGW/Fixes.txt (D008E, D032G, D038G, D343E)
- Airports/EGGW/Sids.txt
- Airports/EGGW/Stars.txt
- Airports/EGKB/Stars.txt
- Airports/EGKK/Stars.txt
- Airports/EGLC/Stars.txt
- Airports/EGLL/Stars.txt
- Airports/EGSC/Stars.txt
- Airports/EGSS/Stars.txt
- Navaids/FIXES_OLD.txt (file retained)

### For #5622:
Replaces EGNR OKTEM 1L STAR with PEPZEL 1L.
Removes EGWU BIG 1F, BIG 3C, BEDEK 1H, BEDEK 1Z STARs.
Adds EGWU SIRIC 1H, SIRIC 1Z STARs.

Liaely commented:
@hazzas-99 I'll work on that tomorrow afternoon.

Liaely commented:
@hazzas-99 Can you assign me to issue #5622?
I'll include a second changelog entry as well, but keep the changes in here.

aidenbok203 commented:
@Liaely Could you please comment on the issue? Unable to assign unless you’ve participated in it.

Liaely commented:
I've changed the EGNR OKTEM 1L with PEPZEL 1L.
For EGWU, I'm not sure about it, but I've removed the BIG 1F, BIG 3C STARs as they're not in the EGLL STARs, but if that's incorrect please let me know.
Also, for EGWU, I've changed BEDEK 1H, BEDEK 1Z with SIRIC 1H, SIRIC 1Z STARs.
